### PR TITLE
Generate macros for TensorFlow nodes with multiple outputs

### PR DESCRIPTION
**Pull request by lesters**

### CODE REVIEW

File: config-model/src/test/java/com/yahoo/searchdefinition/processing/RankingExpressionWithTensorFlowTestCase.java [hunk 1]
```diff
@@ -295,6 +295,19 @@ public void testImportingFromStoredExpressionsWithSmallConstants() throws IOExce
         }
```
Comment (bratseth) [inline config-model/src/test/java/com/yahoo/searchdefinition/processing/RankingExpressionWithTensorFlowTestCase.java]: Could you verify that the stored macro is created and have the expected expression here?

**Review by bratseth — CHANGES_REQUESTED** [review]

**Comment by bratseth** [comment]
One comment, otherwise LGTM!

File: config-model/src/test/java/com/yahoo/searchdefinition/processing/RankingExpressionWithTensorFlowTestCase.java (hunk 1 above)
Comment (lesters) [reply config-model/src/test/java/com/yahoo/searchdefinition/processing/RankingExpressionWithTensorFlowTestCase.java]: You mean like we do in the next test, `testMacroGeneration`?

Comment (bratseth) [reply config-model/src/test/java/com/yahoo/searchdefinition/processing/RankingExpressionWithTensorFlowTestCase.java]: Yes, but in the re-loaded model.
In a real system the model is built from file, then later rebuilt from ZooKeeper at which point the original TensorFlow model is not available. The "importFromStored" tests (only) recreates this situation, so I think one of those need to verify that the macros are present and correct after re-loading from ZooKeeper data.

Comment (lesters) [reply config-model/src/test/java/com/yahoo/searchdefinition/processing/RankingExpressionWithTensorFlowTestCase.java]: Ahh I see, thanks.

**Pull request merged**